functions/src/handle_friend_invitation.ts

```diff
@@ -39,7 +39,7 @@ export const handleFriendInvitation = functions.https.onRequest(
         } catch {
           response.status(401).json(apiError("Bad request"));
         }
-        const { inviteId } = request.body;
+        const { inviteId, token } = request.body;
 
         if (!inviteId) {
```

Comment:
just realized we never checked where this link is being clicked from. let's make the token required. Users will have to sign in to accept invites and we verify the token before proceeding.

Reply:
since the token is always required now, there's no need to check whether the invite type is specifically a link